app/requirements.txt

```diff
@@ -1,3 +1,3 @@
-flask
-psycopg
-SQLAlchemy
\ No newline at end of file
+flask==3.1.1
```

Comment:
pin top level dependencies to make env consistent. do not pin all the dependencies printed by pip freeze because pinning subdependencies makes it hard to figure out what actually needs to change when upgrading packages. this is a general practice, not a hard rule. When do DO want to basically fully specify all the packages that were used for dev, then it might be better to use a dedicated tool like `poetry` which has a `poetry.lock` file.